config/crds/bundle_v1alpha1_componentset.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: componentsets.bundle.gke.io
+spec:
+  group: bundle.gke.io
+  names:
+    kind: ComponentSet
+    plural: componentsets
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
```

Comment:
We might already get validation of all of the typemeta/objectmeta stuff when using CRDs, since it's a generic header, so we might not actually have to specify these.

Reply:
The example at https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/ doesn't include apiVersion/kind/metadata in the openAPIV3Schema validation spec.